tests/signers.py

```diff
@@ -31,16 +32,17 @@ class MockSigner():
                 (contract_address, 'another_method', [arg_1, arg_2])
             ]
         )
-                           
+
     """
+
     def __init__(self, private_key):
         self.signer = Signer(private_key)
         self.public_key = self.signer.public_key
-        
-    async def send_transaction(self, account, to, selector_name, calldata, nonce=None, max_fee=0):
+
+    async def send_transaction(self, account, to, selector_name, calldata, nonce=None, max_fee=1):
```

Comment:
Is the `max_fee` change necessary for testing? If so, should we also change the fee in `MockEthSigner`?

Reply:
do we need a default value if it's already on `send_transactions`? it could get out of sync

Reply:
but what if someone just calls `send_transaction` ?